src/auth/entities/user.entity.ts

```
@@ -34,17 +35,15 @@ export class User {
   email: string;
 
+  @OneToMany(() => Application, (application) => application.user)
+  applications: Application[];
+
   @ApiProperty({
     description: 'Password for the user',
     example: 'hashed_password',
   })
   @Column()
-
   password: string
```

ESLint

[error] Insert `;`

(prettier/prettier)